specifications/api/swagger.yaml

```diff
@@ -1851,6 +1959,85 @@ components:
         state: *walletState
         tip: *blockReference
 
+    ApiActiveSharedWallet: &ApiActiveSharedWallet
+      type: object
+      required:
+        - id
+        - name
+        - account_public_key
+        - account_ix
+        - address_pool_gap
+        - payment_script_template
+        - balance
+        - delegation
+        - state
+        - tip
+      properties:
+        id: *walletId
+        name: *walletName
+        account_public_key: *walletAccountPubkey
+        account_ix: *derivationSegment
+        address_pool_gap: *walletAddressPoolGap
+        payment_script_template: *scriptTemplate
+        delegation_script_template: *scriptTemplate
+        balance: *walletBalance
+        delegation: *ApiWalletDelegation
+        state: *walletState
+        tip: *blockReference
+
+    ApiPendingSharedWallet: &ApiPendingSharedWallet
+      type: object
+      required:
+        - id
+        - name
+        - account_public_key
+        - account_ix
+        - address_pool_gap
+        - payment_script_template
+      properties:
+        id: *walletId
+        name: *walletName
+        account_public_key: *walletAccountPubkey
+        account_ix: *derivationSegment
+        address_pool_gap: *walletAddressPoolGap
+        payment_script_template: *scriptTemplate
+        delegation_script_template: *scriptTemplate
+
+    ApiSharedWallet: &ApiSharedWallet
+      type: object
+      oneOf:
+        - title: Pending shared wallet (collecting account public keys for each co-signer)
+          <<: *ApiPendingSharedWallet
+        - title: Active shared wallet (account public keys for each co-signers is collected)
+          <<: *ApiActiveSharedWallet
+
+    ApiSharedWalletPostData: &ApiSharedWalletPostData
+      type: object
+      required:
+        - name
+        - account_public_key
+        - account_ix
```

Comment:
Let's avoid diminutives in the API as much as possible. `account_ix` --> `account_index` (similar to how we have `output_index`, `transaction_index` etc..)

Reply:
done